libs/GameSystems/CMakeLists.txt

```diff
@@ -17,9 +30,25 @@ target_include_directories(${TARGET}
     PRIVATE $(SRC_ROOT)
 )
 
+target_include_directories(${TARGET_CLIENT}
```

Comment:
If this issue is merged after the issue #114 please refer to the new upper architecture for link libraries